File: CSharp/Contests2025/ABC409/A.cs
class A
{
	static void Main() => Console.WriteLine(Solve() ? "Yes" : "No");
	static bool Solve()
	{
		var n = int.Parse(Console.ReadLine());
		var t = Console.ReadLine();
		var a = Console.ReadLine();

		return t.Zip(a, (x, y) => x == 'o' && y == 'o').Any(b => b);
	}
}

File: Java/Contests2025/ABC409/src/A.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class A {

	static BufferedReader in = new BufferedReader(new InputStreamReader(System.in));

	static String readLine() {
		try {
			return in.readLine();
		} catch (IOException ex) {
			return "";
		}
	}

	public static void main(String[] args) {
		System.out.println(solve() ? "Yes" : "No");
	}

	static boolean solve() {
		var n = Integer.parseInt(readLine());
		var t = readLine();
		var a = readLine();

		for (int i = 0; i < n; i++)
			if (t.charAt(i) == 'o' && a.charAt(i) == 'o')
				return true;
		return false;
	}
}
